### ISSUE TITLE
Corrección TP#1

### ISSUE THREAD

**Issue by nataliapl** [ccalvoromero/AyED-TP-01#1]
Trabajo Aprobado! Te compiló sin usar una biblioteca para manejar cadenas? Yo la tuve que agregar. Qué IDE usas?

Saludos
Natalia.

**Comment by ccalvoromero** [comment]
Sí, compiló, todo normal. Recién me doy cuenta que me faltó la biblioteca string.h , así que, raro. Uso el Dev C-C++.

Saludos
